Moodlet/app/src/main/java/com/cmput3owo1/moodlet/activities/MoodEditorActivity.java

```diff
@@ -4,6 +4,7 @@
 
 import android.content.Intent;
 import android.os.Bundle;
+import android.widget.Toast;
```

Comment:
Is this import used?

Reply:
Nope. I forgot to get rid of that after testing.